ida-customizations/ida-course/LPs/week-1/index.md

```diff
@@ -40,7 +40,7 @@ In this chapter, you will:
 
 **This week, make sure to submit your quiz once you feel ready to test your knowledge.**
 
-The quiz will be available until the Final Exam is launched on July 6th.
+The quiz will be available until the Final Exam is launched on January 4th, 2024, but it is recommended to attempt it immediately after completing the Week 1 content.
```

Comment:
Adapt based on the Final Exam split - if the quiz is part of the Final Exam it should stay open until Feb 1st.

Reply:
Yes, Ibo confirmed that it will stay open. I'll make the change.